app/models/megaphone/episode.rb

```diff
@@ -16,15 +19,19 @@ class Episode < Integrations::Base::Episode
 
     CREATE_ATTRIBUTES = CREATE_REQUIRED + %i[pubdate pubdate_timezone author link explicit draft
       subtitle summary background_image_file_url background_audio_file_url pre_count post_count
-      insertion_points guid expected_adhash original_filename original_url
-      episode_number season_number retain_ad_locations advertising_tags ad_free]
+      insertion_points guid expected_adhash original_filename episode_number season_number
+      retain_ad_locations ad_free episode_type clean_title]
 
     UPDATE_ATTRIBUTES = CREATE_ATTRIBUTES
 
     # All other attributes we might expect back from the Megaphone API
     # (some documented, others not so much)
-    OTHER_ATTRIBUTES = %i[id podcast_id created_at updated_at status
-      download_url audio_file_processing audio_file_status audio_file_updated_at pre_offset post_offset]
+    OTHER_ATTRIBUTES = %i[id podcast_id created_at updated_at status download_url
+      audio_file_processing audio_file_status audio_file_updated_at pre_offset post_offset
+      image_file audio_file size duration uid original_url bitrate samplerate channel_mode vbr
+      id3_file id3_file_processing id3_file_size spotify_identifier custom_fields content_rating
+      promotional_content podcast_title network_id podcast_author podcast_itunes_categories
+      episode_video_thumbnail_url main_feed rss_status spotify_status cuepoints advertising_tags]
```

Comment:
There's a bunch of semi-documented attributes that come back - go ahead and capture them all